modules/rgbd/src/hash_tsdf.hpp

```diff
@@ -33,16 +33,8 @@ class HashTSDFVolume : public Volume
     int volumeUnitResolution;
     float volumeUnitSize;
     bool zFirstMemOrder;
-};
-
-struct VolumeUnit
-{
-    VolumeUnit() : pVolume(nullptr){};
-    ~VolumeUnit() = default;
-
-    cv::Ptr<TSDFVolume> pVolume;
-    cv::Vec3i index;
-    bool isActive;
+    Point3i volDims;
+    Vec4i volStrides;
```

Comment:
`volStrides` is implementation-dependent, it's better to move it to CPU class

Reply:
done